lib/pages/profile_info/profile_info.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:fluffychat/pages/profile_info/profile_info_view.dart';
+import 'package:fluffychat/widgets/matrix.dart';
+import 'package:flutter/material.dart';
+import 'package:matrix/matrix.dart';
+
+class ProfileInfo extends StatefulWidget {
```

Comment:
not clearly understand this widget,

Reply:
I renamed it to ProfileInfoPage. It's the screen that is displayed on mobile